### PR TITLE
Read settings from environment, fixes #74

### LINKED ISSUE #74: Setup different settings files (development, production)
Use DJANGO_SETTINGS_MODULE as corresponding environment variable 

* remove EMAIL_BACKEND and setup a production ready mail server
* set DEBUG to false
* django won't handle static files anymore

### PR DESCRIPTION
**Pull request by svenseeberg**
Relevant settings can now be set with environment variables. If a environment variable is not set, the default value from `settings.py` is used. The `wsgi.py` needs to read the call environment variables and turn them into os.environ variables. An example config for Apache2 is included.

Fixes: #74

### CODE REVIEW

File: examples/apache2-vhost.conf [hunk 1]
```diff
@@ -0,0 +1,40 @@
+WSGIScriptAlias / /opt/integreat-cms/src/backend/wsgi.py
```
Comment (timoludwig) [inline examples/apache2-vhost.conf:1]: I'm not sure if "examples" is a good directory name here, maybe rather "apache-conf"/"prod-conf"/"example-conf" or similar?

**Review by timoludwig — APPROVED** [review]
Looks good to me! :+1: :tada:

File: examples/apache2-vhost.conf (hunk 1 above)
Comment (svenseeberg) [reply examples/apache2-vhost.conf:1]: My idea here was to potentially collect other example config files here as well. What about `example-configs` ?

Comment (timoludwig) [reply examples/apache2-vhost.conf:1]: ok, sounds good :+1:

**Comment by ulliholtgrave** [comment]
Please wait a second till you merge the PR. I am currently setting up the system on a clean system and I want to try, if everything works out :)

**Comment by ulliholtgrave** [comment]
Okay, everything works as expected :) 👍

**Pull request merged**